scripts/release-notes.py

```diff
@@ -76,7 +76,7 @@ def main(token, repo, num_commits, exclude_dependabot):
 
         author_url = commit['author']['html_url']
         msg_lines = commit['commit']['message'].split('\n')
-        msg = msg_lines[0]
+        msg = msg_lines[0].capitalize()
```

Comment:
This is the only remaining change, a bit of a nit, so happy to just close this PR if you think it's not worthwhile 😅